pages/common/pre-commit.md

```diff
@@ -0,0 +1,20 @@
+# pre-commit
+
+> Manage and maintain multi-language pre-commit hooks.
```

Comment:
> How about this? 'multi-language'&mdash;although I see you took verbatim from the docs&mdash;is not obvious to new users (like me ;)).
> ```suggestion
> > Create Git hooks that get run before a commit using common programming languages.
> ```

I like that suggestion, but it seems like it could be even shorter:
```
Create Git hooks that get run before a commit.
```

Reply:
The reason I added 'using common programming languages' is: that was my impression of what 'multi-language' means in this context, because it doesn't seem perfectly obvious.  Do you have a better explanation of that, Hugh?

Reply:
I think that the shortened one without the mention of multi-language, etc. is probably cleanest. 👍🏻

Reply:
That it will be, then.
```
Create Git hooks that get run before a commit.
```